src/docs/cookbook/testing/widget/tap-drag.md

```diff
@@ -232,6 +238,8 @@ void main() {
 }
 
 class TodoList extends StatefulWidget {
+  const TodoList({Key? key}) : super(key: key);
```

Comment:
fwiw, I'd add `use_key_in_widget_constructors: false` to the linter rules above so you don't need to introduce this cconstructor here.

Reply:
@sfshaza2 if @miquelbeltran isn't around to make these changes, feel free to land as is.